api/adobe_vendor_id.py

```diff
@@ -1092,3 +1093,93 @@ def explain(self, output, libraries, website):
         output.write("Website: %s\n" % website)
         output.write(" Short name: %s\n" % short_name)
         output.write(" Short Client Token secret: %s\n" % secret)
+
+
+class ShortClientTokenLibraryConfigurationScript(Script):
+
+    @classmethod
+    def arg_parser(cls):
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            '--website-url', 
+            help="The URL to this library's patron-facing website (not their circulation manager), e.g. \"https://nypl.org/\". This is used to uniquely identify a library.",
+            required=True,
+        )
```

Comment:
I use the patron-facing URL, rather than some other method for identifying which library you're editing, because patron-facing URL is what I ask for when setting up a SCT integration for a library.